WoWsShipBuilder.Web/Components/ShipStatsContainer.razor

```diff
@@ -163,14 +195,19 @@
             var elementIds = new List<string>();
             if (ViewModel.PreviousShip is not null)
             {
-                elementIds.Add(ViewModel.CurrentShipIndex + "_" + ViewModel.PreviousShip.Index);
+                elementIds.Add(ViewModel.CurrentShipIndex + "_" + ViewModel.PreviousShip.Index + "_" + CurrentTabId);
             }
             if (ViewModel.NextShips is not null)
             {
-                elementIds.AddRange(ViewModel.NextShips.Select(ship => ViewModel.CurrentShipIndex + "_" + ship.Index));
+                elementIds.AddRange(ViewModel.NextShips.Select(ship => ViewModel.CurrentShipIndex + "_" + ship.Index + "_" + CurrentTabId));
             }
+            
             StateHasChanged();
-            await MouseEventInterop.PreventMiddleClickDefaultBatched(elementIds);
+            
+            if (elementIds.Any())
+            {
+                await MouseEventInterop.PreventMiddleClickDefaultBatched(elementIds);
```

Comment:
This does no longer work properly. It only works when opening a tab the first time, after that it does not register the listener again